Vagrantfile

```diff
@@ -245,6 +246,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       os.ssh_username = user = voc['ssh_user']|| creds['OSSshUser']  || "root"           # login for the VM instance
       os.server_name  = ENV['OS_HOSTNAME']    || vagrant_openshift_config['instance_name'] # name for the instance created
       full_provision(override.vm, user)
+      floating_ip_pool = creds['OSFloatingIPPool'] || ENV['OS_FLOATING_IP_POOL']
```

Comment:
As far as I'm reading docs, `floating_ip` takes an IP or `:auto`. Using a static IP does not make sense in a cluster config but given cluster config is not possible with anything but virtualbox at the moment, I assume people are usually running a single instance to contain master and minion. So one could want to specify a static IP address in that property. And I don't think  something like `:"10.10.10.10"` makes sense. Also `nil.to_sym` in your example could be a problem. I'll update PR to have 2 lines less.